_data/su_networks.yml

```diff
@@ -65,12 +65,12 @@
       na:
         stable:
           starttls: direct TLS only
-    - name: Oragono Testnet
+    - name: Ergo Testnet
```

Comment:
Could you please move this to the start of the testnet list? (alphabetical order, etc)